public/javascripts/SVLabel/src/SVLabel/onboarding/OnboardingStates.js

```diff
@@ -73,8 +73,8 @@ function OnboardingStates (compass, mapService, statusModel, tracker) {
                 {
                     "name": "arrow-1a",
                     "type": "arrow",
-                    "x": 9710,
-                    "y": -325,
+                    "x": 9730,
```

Comment:
why are all the x/y positions being modified slightly?

Reply:
The custom panos didn't line up perfectly with the previous arrows when I was trying to set the pano's heading. So I adjusted them slightly to make it match the previous tutorial perfectly.